assets/js/timetree.js

```diff
@@ -205,8 +216,14 @@ function TreeGraph({nodes, links, centuries}) {
     // .force("link", d3.forceLink(links).distance(30).strength(link => {
       // return 1;
     // }))
-    .force("y", d3.forceY().y(node => centuryY(node)).strength(0.8))
-    .on("tick", ticked);
+    .force("y", d3.forceY().y(node => centuryY(node)).strength(0.8));
+    // .on("tick", ticked);
+
+  // run simulation for 300 ticks without animation
+  simulation.tick(300);
+  // only position once after simulation has run
+  simulation.on("tick", ticked);
+  simulation.tick();
```

Comment:
Are these changes related to lighthouse or did they slip in?

Reply:
I implemented this to address some loading speed & related problems that lighthouse flagged

Reply:
Nice!

Reply:
@rlskoeser is "ticks" what makes it generative?

Reply:
I should probably revise, but this is normally how you redraw the graph after every iteration of the simulation applying forces and calculating positions.